README.md

```diff
@@ -264,7 +267,7 @@ docker login ghcr.io -u <your-github-username> -p <your-github-personal-access-t
 
 export KEYSTORE_PASSWORD=<your-keystore-password>
 export IMAGE_TAG=1.0
-./build.sh $KEYSTORE_PASSWORD $REGISTRY_NAME $IMAGE_TAG ${NGINX_VERSION} ${OTEL_MODULE_VERSION}
+./build.sh $KEYSTORE_PASSWORD $IMAGE_TAG ${NGINX_VERSION} ${OTEL_MODULE_VERSION} $REGISTRY_NAME

```

Comment:
I find `build.sh` is too small a name for what it does. I suggest renaming `build.sh`  to something more accurate, such as `build-piggymetrics-microservices.sh`. What do you think?